src/res_string.ml

```diff
@@ -1,4 +1,22 @@
-let hexTable = [|'0'; '1'; '2'; '3'; '4'; '5'; '6'; '7'; '8'; '9'; 'a'; 'b'; 'c'; 'd'; 'e'; 'f'|]
+let hexTable =
+  [|
+    '0';
+    '1';
+    '2';
+    '3';
+    '4';
+    '5';
+    '6';
+    '7';
+    '8';
+    '9';
+    'a';
+    'b';
+    'c';
+    'd';
+    'e';
+    'f';
+  |]
```

Comment:
If you like to keep this as a one-liner, you can use `[@@ocamlformat "disable"]` here.